pkgs/by-name/za/zarf/package.nix

```
@@ -53,8 +82,11 @@ buildGoModule rec {
   meta = with lib; {
     description = "DevSecOps for Air Gap & Limited-Connection Systems. https://zarf.dev";
```

```suggestion
    description = "DevSecOps for air-gapped / limited-connectivity systems";
```
Now redundant with `meta.homepage`. More compliant with the style guide for `meta.description`.